docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -140,6 +145,7 @@ occurred
 |type |String a|Data type on which the rule is based:
 
 * `eql`: EQL query (see {ref}/eql.html[Event Query Language]).
+* `esql`: ES|QL query (see {ref}/esql.html[Elasticsearch Query Language]).
```

Comment:
```suggestion
* `esql`: ES\|QL query (see {ref}/esql.html[Elasticsearch Query Language]).
```
In the HTML preview, this line renders as:
 * `esql`: ES

 I think we need to escape the pipe character since this content is inside a table.